tmva/tmva/inc/TMVA/DNN/Net.h

```diff
@@ -189,8 +189,8 @@ TNet<Architecture_t, Layer_t>::TNet(size_t batchSize,
       AddLayer(other.GetLayer(i).GetWidth(),
                other.GetLayer(i).GetActivationFunction(),
                other.GetLayer(i).GetDropoutProbability());
-      fLayers[i].GetWeights() = (TMatrixT<Double_t>) other.GetLayer(i).GetWeights();
-      fLayers[i].GetBiases()  = (TMatrixT<Double_t>) other.GetLayer(i).GetBiases();
+      fLayers[i].GetWeights() = (TMatrixT<Scalar_t>) other.GetLayer(i).GetWeights();
```

Comment:
The type parameter is in some places called `Scalar_t`, in some places `AFloat` and in some places `AReal`. Since we're changing this anyway, maybe we want to ensure some consistency.